client/app/bundles/shared/components/Blockquote.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import css from './Blockquote.scss';
+
+type Props = {
+  text?: string,
+  author?: string,
+};
+
+export default class Blockquote extends React.Component<Props> {
+  render() {
+    const { text, author } = this.props;
+    const textClassNames = `${css.author} ${css.text}`;
+    const authorClassNames = `${css.author}`;
+    return (
+      <div>
```

Comment:
Let's give the container a class here, and perhaps use a `<blockquote>`! All container properties should be applied to this class as well (`background-color`, `padding`, etc.). Currently, the quote is split into two parts, the text and author, which is great, but they each have container-like properties.